src/me/StevenLawson/TotalFreedomMod/Commands/Command_smite.java

```diff
@@ -10,13 +10,13 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
-@CommandParameters(description = "Someone being a little bitch? Smite them down...", usage = "/<command> [playername]")
+@CommandParameters(description = "Someone being a little bitch? Smite them down...", usage = "/<command> [playername] [reason]")
```

Comment:
@JeromSar So it should. I was only looking at the changes that were made, thats why I missed it :P